cwms-data-api/src/test/java/cwms/cda/api/LevelsControllerTestIT.java

```diff
@@ -55,6 +55,62 @@ public class LevelsControllerTestIT extends DataApiTestIT {
 
     public static final String OFFICE = "SPK";
 
+    @Test
+    void test_location_level() throws Exception {
+        createLocation("level_as_single_value", true, OFFICE);
+        String levelId = "level_as_single_value.Stor.Ave.1Day.Regulating";
+        ZonedDateTime time = ZonedDateTime.of(2023, 6, 1, 0, 0, 0, 0, ZoneId.of("America/Los_Angeles"));
+        LocationLevel level = new LocationLevel.Builder(levelId, time)
+                .withOfficeId(OFFICE)
+                .withConstantValue(1.0)
+                .withLevelUnitsId("ac-ft")
+                .build();
+            CwmsDataApiSetupCallback.getDatabaseLink().connection(c -> {
+                DSLContext dsl = dslContext((Connection) c, OFFICE);
+                LocationLevelsDaoImpl dao = new LocationLevelsDaoImpl(dsl);
+                dao.storeLocationLevel(level, level.getLevelDate().getZone());
+            });
+
+        //Read level without unit
+        given()
+            .accept(Formats.JSONV2)
```

Comment:
@adamkorynta, @rma-rripken FYI for future tests I suggest following this formatting. It makes the given/when/then conditions more obvious when skimming.

Might not matter to a lot but it certainly helps me reading through things.